components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/fragments/policy-add/policy-add.hbs

```diff
@@ -140,8 +146,104 @@
             </select>
         </div>
     </div>
-</form>
+    {{#if @queryParams.subscription}}
+    <div class="page-header">
+        <h2><br></h2>
+        <p class="lead">{{i18n "burst.control.rate.limiting"}}</p>
+    </div>
+
+    <div class="form-group">
+        <label class="control-label col-sm-3">{{i18n "Request.Count"}} </label>
+        <div class="col-sm-6">
```

Comment:
I haven't intended to achieve different grid ratios based on the device. (i.e col-sm-6 col-md-4 col-lg-3). I intended to use same grid ratios for all devices. (i.e col-sm-6 col-md-6 col-lg-6). So
"col-lg-6 col-md-6 col-sm-6" = "col-sm-6". Thats why i used only col-sm-6. 
If there is any particular reason to use different grid ratios, then we can change this